source/rrRoadRunner.cpp

```diff
@@ -5011,6 +5014,23 @@ void RoadRunner::saveSelectionVector(std::ostream& out, std::vector<SelectionRec
 void RoadRunner::loadState(std::string filename)
 {
 	std::ifstream in(filename, iostream::binary);
+	if (!in)
```

Comment:
Have you closed this file once I/O was done?

Reply:
The dtor of std::ifstream will take care of closing the file, I think.